submissions/keiran-vien/scripts/article.js

```diff
@@ -23,6 +23,14 @@
   // TODO:DONE Set up a DB table for articles.
   Article.createTable = function(callback) {
     webDB.execute(
+      'DROP TABLE articleData;', // what SQL command do we run here inside these quotes?
+      function(result) {
+        console.log('fuck the table', result);
+        if (callback) callback();
+      }
+    );
+
+    webDB.execute(
       'CREATE TABLE articleData (id INTEGER PRIMARY KEY, title VARCHAR, author VARCHAR, authorUrl VARCHAR, category VARCHAR, publishedOn VARCHAR, body VARCHAR);', // what SQL command do we run here inside these quotes?
```

Comment:
You could just use 'CREATE TABLE IF NOT EXISTS articleData' here and you wouldn't be dropping the table every time.